### ISSUE TITLE
installation requires manual compilation

### ISSUE THREAD

**Issue by FND** [saifelse/coffeelint-no-implicit-returns#6]
after installing this module via npm and adjusting the configuration, CoffeeLint reports "Error loading coffeelint-no-implicit-returns"

that's because `package.json` states `"main": "index.js"`, yet npm only installs `index.coffee`

manually compiling `index.coffee` to `index.js` works, but is rather inconvenient

**Comment by BernieSumption** [comment]
I've created a pull request that fixes this issue: https://github.com/saifelse/coffeelint-no-implicit-returns/pull/8

If the maintainer can merge it and republish I'd be grateful, but in the mean time anyone who wants this fix can point their package.json at my GitHub instead:

```js
"devDependencies": {
    // ...
    "coffeelint-no-implicit-returns":
        "BernieSumption/coffeelint-no-implicit-returns#9c8f4dfc16"
    // ...
}
```

I'll keep my fork live at least until this PR is merged, but be aware that if you use the above link, it may break in the future once the repo is no longer required.

Cheers,

Bernie      :o)

**Comment by saifelse** [comment]
Apologies for the delay. Merged (cdc7bc1b99e3e67a2cdfcf9876bbbbf7ef1c857d) and republished (06c6614c99d308065c0384dbccb190a1bbf58459)!
